rootfs/usr/share/inputplumber/schema/capability_map_v2.json

```diff
@@ -0,0 +1,1257 @@
+        "SW_JACK_PHYSICAL_INSERT",
+        "SW_VIDEOOUT_INSERT",
+        "SW_CAMERA_LENS_COVER",
+        "SW_KEYPAD_SLIDE",
+        "SW_FRONT_PROXIMITY",
+        "SW_ROTATE_LOCK",
+        "SW_LINEIN_INSERT",
+        "SW_MUTE_DEVICE",
+        "SW_PEN_INSERTED",
+        "SW_MACHINE_COVER",
+        "MSC_SERIAL",
+        "MSC_PULSELED",
+        "MSC_GESTURE",
+        "MSC_RAW",
+        "MSC_SCAN",
+        "MSC_TIMESTAMP",
+        "LED_NUML",
+        "LED_CAPSL",
+        "LED_SCROLLL",
+        "LED_COMPOSE",
+        "LED_KANA",
+        "LED_SLEEP",
+        "LED_SUSPEND",
+        "LED_MUTE",
+        "LED_MISC",
+        "LED_MAIL",
+        "LED_CHARGING",
+        "REP_DELAY",
+        "REP_PERIOD",
+        "SND_CLICK",
+        "SND_BELL",
+        "SND_TONE"
+      ]
+    },
+    "EventType": {
+      "oneOf": [
+        {
+          "description": "A bookkeeping event. Usually not important to applications.",
+          "type": "string",
+          "enum": [
+            "SYNC"
+          ]
+        },
+        {
+          "description": "A key changed state. A key, or button, is usually a momentary switch (in the circuit sense). It has two states: down, or up. There are events for when keys are pressed (become down) and released (become up). There are also \"key repeats\", where multiple events are sent while a key is down.",
+          "type": "string",
+          "enum": [
+            "KEY"
+          ]
+        },
+        {
+          "description": "Movement on a relative axis. There is no absolute coordinate frame, just the fact that there was a change of a certain amount of units. Used for things like mouse movement or scroll wheels.",
+          "type": "string",
+          "enum": [
+            "REL"
+          ]
+        },
+        {
+          "description": "Movement on an absolute axis. Used for things such as touch events and joysticks.",
+          "type": "string",
+          "enum": [
+            "ABS"
+          ]
+        },
+        {
+          "description": "Miscellaneous events that don't fall into other categories. For example, Key presses may send `MscSCAN` events before each KEY event",
+          "type": "string",
+          "enum": [
+            "MSC"
+          ]
+        },
+        {
+          "description": "Change in a switch value. Switches are boolean conditions and usually correspond to a toggle switch of some kind in hardware.",
+          "type": "string",
+          "enum": [
+            "SW"
+          ]
+        },
+        {
+          "description": "An LED was toggled.",
+          "type": "string",
+          "enum": [
+            "LED"
+          ]
+        },
+        {
+          "description": "A sound was made.",
+          "type": "string",
+          "enum": [
+            "SND"
+          ]
+        },
+        {
+          "description": "There are no events of this type, to my knowledge, but represents metadata about key repeat configuration.",
```

Comment:
This seems a little too narrative.

Reply:
I pulled this comment from the evdev library, but I will update.